crates/symbolicator-js/src/utils.rs

```diff
@@ -713,6 +715,12 @@ mod tests {
         assert_eq!(is_in_app(abs_path, filename), Some(true));
         assert_eq!(is_in_app_faithful(abs_path, filename), Some(true));
 
+        let abs_path = "webpack:///@sentry/browser/esm/helpers.js";
```

Comment:
Please use this as your test case "webpack:///foo/bar/src/App.jsx"`

When you have `@sentry` in the path it hints at an installed package. The actual path from the JSON event contained `node_modules`: `"webpack:///foo/bar/node_modules/@sentry/browser/esm/helpers.js"`